interfaces/builtin/fwupd.go

```diff
@@ -254,6 +258,28 @@ func (iface *fwupdInterface) AppArmorPermanentSlot(spec *apparmor.Specification,
 	// classic, slot side can be system or application
 	if !implicitSystemPermanentSlot(slot) {
 		spec.AddSnippet(fwupdPermanentSlotAppArmor)
+
+		// Allow mounting boot partition to snap-update-ns
+		emit := spec.AddUpdateNSf
+		target := "/boot"
+		source := "/var/lib/snapd/hostfs" + target
+		emit("  # Read-write access to %s\n", target)
+		emit("  mount options=(rbind) %s/ -> %s/,\n", source, target)
+		emit("  umount %s/,\n\n", target)
```

Comment:
Nit-pick: since this isn't a loop any more, it is arguably clearer to not use the target and source variables and just hard code in the emits. Not a blocker. No need to adjust unless changing other things.